src/main/java/org/smartregister/pathevaluator/action/ActionHelper.java

```diff
@@ -141,7 +149,9 @@ public static ResourceType getResourceType(Action action) {
 
 			case GLOBAL_TASK:
 				return PathEvaluatorLibrary.getInstance().getTaskProvider().getAllTasks(resource.getId());
-			
+
+			case INVENTORY:
+//				return PathEvaluatorLibrary.getInstance().getStockProvider().getStocksAgainstServicePointId(resource.getId()); //TODO
```

Comment:
@githengi  is this okay to fetch all stock items from a service point id?

Reply:
Yes